src/interfaces/swig/shogun.i

```diff
@@ -295,6 +295,7 @@ PUT_ADD(CEvaluation)
 PUT_ADD(CSVM)
 PUT_ADD(CMeanFunction)
 PUT_ADD(CLikelihoodModel)
+PUT_ADD(ParameterObserver)
```

Comment:
Do we ever need to put a parameter observer?

Reply:
Not sure. If we use the `subscribe` method of `SGObject` it may not be needed.

Reply:
ok then remove this here